### PR TITLE
MFI main application code

### PR DESCRIPTION
**Pull request by ankitadattatraymore**

### CODE REVIEW

File: src/main/java/mfi/application/demo/MFIApp.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package mfi.application.demo;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.ComponentScan;
+
+
+
+@ComponentScan(basePackages = {"com.example","mfi.application"})
```
Comment (READIGenz) [inline src/main/java/mfi/application/demo/MFIApp.java:9]: please use better package name

File: src/main/java/mfi/application/demo/UrlConstants.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
+package mfi.application.demo;
```
Comment (READIGenz) [inline src/main/java/mfi/application/demo/UrlConstants.java:1]: please use a better package name

File: src/test/java/mfi/application/demo/ConsumerUiApplicationTests.java [hunk 3]
```diff
@@ -0,0 +1,13 @@
+package mfi.application.demo;
```
Comment (READIGenz) [inline src/test/java/mfi/application/demo/ConsumerUiApplicationTests.java:1]: pls use a better package name

**Review by READIGenz — CHANGES_REQUESTED** [review]
overall PR looks good changes needed on the package name

File: src/main/java/mfi/application/demo/UrlConstants.java (hunk 2 above)
Comment (ankitadattatraymore) [reply src/main/java/mfi/application/demo/UrlConstants.java:1]: should I keep package name as mfi.main.app

File: src/main/java/mfi/application/demo/HomeController.java [hunk 4]
```diff
@@ -0,0 +1,104 @@
+package mfi.application.demo;
```
Comment (READIGenz) [inline src/main/java/mfi/application/demo/HomeController.java:1]: pls use a other package name apart from demo

File: src/main/java/mfi/application/demo/UrlConstants.java (hunk 2 above)
Comment (READIGenz) [reply src/main/java/mfi/application/demo/UrlConstants.java:1]: Yes, looks relevant.
or just mfi.app is ok as well.

**Review by READIGenz — APPROVED** [review]
approved with changes

File: src/main/java/mfi/application/demo/HomeController.java (hunk 4 above)
Comment (ankitadattatraymore) [reply src/main/java/mfi/application/demo/HomeController.java]: done.

**Pull request merged**